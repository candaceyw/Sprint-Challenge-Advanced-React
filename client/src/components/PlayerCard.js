import React, { Component } from 'react'

 class PlayerCard extends Component {
    render() {
        return (
            <div className="cardWrapper">
            <div className="playerCard">
            <h2>{this.props.name}</h2>
            <p>Country: {this.props.country}</p>
            <p>Searches: {this.props.searches}</p>
          </div>
          </div>
           
        )
    }
}

export default PlayerCard;