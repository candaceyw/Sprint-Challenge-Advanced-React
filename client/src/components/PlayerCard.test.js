import React from 'react'
import PlayerCard from './PlayerCard'

import { render, fireEvent, waitForElement, getByText } from '@testing-library/react'
import renderer from 'react-test-renderer';
// import axiosMock from 'axios'







describe('PlayerCard', () => {
    test('snapshot renders', () => {
      const component = renderer.create(<PlayerCard />);
      let tree = component.toJSON();
      expect(tree).toMatchSnapshot();
    });
  });