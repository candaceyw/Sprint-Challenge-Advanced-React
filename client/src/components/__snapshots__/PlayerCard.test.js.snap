// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlayerCard snapshot renders 1`] = `
<div
  className="cardWrapper"
>
  <div
    className="playerCard"
    data-testid="playerCard"
  >
    <h2 />
    <p>
      Country: 
    </p>
    <p>
      Searches: 
    </p>
  </div>
</div>
`;
